quickstart/docker-compose.yaml

```diff
@@ -0,0 +1,115 @@
+version: '3.7'
```

Comment:
move to it's own subdirectory (`oidc-provider`) here https://github.com/ory/kratos/tree/master/contrib

Reply:
Removed, see ory/kratos#538